### ISSUE TITLE
Add checkStyle and pmd for easy conformance to project code style

### ISSUE THREAD

**Issue by pareshchouhan** [vickychijwani/quill#118]
Add [checkStyle](https://docs.gradle.org/current/userguide/checkstyle_plugin.html) so we can have consistent code style across the project.  
[pmd](https://docs.gradle.org/current/userguide/pmd_plugin.html) is yet another plugin which performs quality checks on java source files.  
need a thumbs up so I can add this ;)

**Comment by vickychijwani** [comment]
Hi @pareshchouhan! Sorry for the late response. I would love to include this.

Does this plugin run automatically as part of every build in Android Studio? I'd prefer to not have it run like that as it would slow down builds somewhat. Can this instead be set to run using Github's PR hooks with Travis CI / Circle CI?

**Comment by pareshchouhan** [comment]
yes they can be set to run independently of the main build so this won't slow down the build :)

**Comment by vickychijwani** [comment]
Sounds good :+1:
